stories/Components/UIcomponents/Cards/ContentCard/contentcard.scss

```diff
@@ -165,6 +172,29 @@
       margin-left: 0;
     }
   }
+  &.yellow {
+    .card-thumbnail__image::before {
+      @include background-image(318deg, $color-yellow, transparent 80%);
+    }
+  }
+  &.red {
+    .card-thumbnail__image::before {
+      @include background-image(318deg, $color-red, transparent 80%);
+    }
+  }
+
+  &.green {
+    .card-thumbnail__image::before {
+      @include background-image(318deg, $color-green, transparent 80%);
+    }
+  }
+
+  &.blue {
+    .card-thumbnail__image::before {
+      @include background-image(318deg, $color-azure, transparent 80%);
+    }
+  }
+  }
```

Comment:
This can be similar written as mentioned in the above comment.

Reply:
Done